docker-compose.yaml

```diff
@@ -29,6 +29,82 @@ services:
     depends_on:
       - postgres
 
+  zoo1:
+    image: confluentinc/cp-zookeeper:7.3.2
+    platform: linux/amd64
+    hostname: zoo1
+    container_name: zoo1
+    ports:
+      - "2181:2181"
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_SERVER_ID: 1
+      ZOOKEEPER_SERVERS: zoo1:2888:3888
+    networks:
+      - app-network
+
+
+  kafka1:
```

Comment:
do we really need **3** brokers?

Reply:
The internet says: `Scalability: Multiple brokers allow Kafka to handle more data and serve more producers and consumers. Data is spread across brokers by dividing topics into partitions, each of which is managed by a separate broker. Fault Tolerance: If one broker fails, other brokers in the cluster can take over.` 
But the system will work with 1 broker as well. We can decrease this number. I picked 3 because itt was recommended by the course

Reply:
I read that the optimal way would be to have 1 for development, 3 for prod; but actually we can leave it as it is, cause having several brokers is kinda one of the Kafka`s features